app/src/main/java/org/mozilla/fenix/ext/Glean.kt

```diff
@@ -1,31 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-package org.mozilla.fenix.ext
-
-import mozilla.telemetry.glean.private.TimingDistributionMetricType
-
-/**
- * A reimplementation of [TimingDistributionMetricType.measure] that address unintuitive
- * issues around non-local returns: see https://bugzilla.mozilla.org/show_bug.cgi?id=1699505.
- * This should be removed once that bug is resolved. That method's kdoc is as follows:
- *
- * Convenience method to simplify measuring a function or block of code.
- *
- * If the measured function throws, the measurement is canceled and the exception rethrown.
- */
-@Suppress("TooGenericExceptionCaught")
-fun <U> TimingDistributionMetricType.measureNoInline(funcToMeasure: () -> U): U {
```

Comment:
Should we maybe keep this around in case we need it again?

Reply:
Good catch – however, this was actually working around [a bug in Glean](https://bugzilla.mozilla.org/show_bug.cgi?id=1699505) that has since been fixed so the method is unnecessary and we should keep it removed: calling Glean's `measure` directly is the proper solution and we just hadn't updated our code to use the new method yet.